charts/ccd-definition-store-api/values.yaml

```diff
@@ -38,6 +38,7 @@ java:
     ELASTIC_SEARCH_DYNAMIC: true
     ELASTIC_SEARCH_CASE_INDEX_NAME_FORMAT: '%s_cases'
     ELASTIC_SEARCH_CASE_INDEX_MAPPING_FIELDS_LIMIT: 8000
+    ELASTIC_SEARCH_AUTOMATIC_INDICES_CREATION_ENABLED: true
```

Comment:
thinking better about it, do we really need this now that this is triggered manually via an endpoint?